README.md

```diff
@@ -1,5 +1,10 @@
-# ATLAS animal_data Service Template \[v0.1.1\]
+# ATLAS animal_data Service Template \[v0.2\]
```

Comment:
Always use 3 digit version numbers. I this case, 0.2.0. Please also change all the links containing 0.1.1 to 0.2.0 or the links will break on release.